WebKitBrowser/CMakeLists.txt

```diff
@@ -15,33 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.3)
-find_package(WPEFramework)
-
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../cmake/")
-
-include(CmakeHelperFunctions)
-
-# Library installation section
-string(TOLOWER ${NAMESPACE} STORAGE_DIRECTORY)
-
-if(BUILD_REFERENCE)
-    add_definitions(-DBUILD_REFERENCE=${BUILD_REFERENCE})
-endif()
+project(WebKitBrowser)
 
 set(PLUGIN_NAME WebKitBrowser)
 set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
 
-find_package(ProxyStubGenerator REQUIRED)
-find_package(JsonGenerator REQUIRED)
-
-ProxyStubGenerator(INPUT ${CMAKE_CURRENT_SOURCE_DIR}/interfaces)
-JsonGenerator(DOCS CODE STUBS
-  INPUT ${CMAKE_CURRENT_SOURCE_DIR}/WebKitBrowserPlugin.json
-  IFDIR  ${CMAKE_CURRENT_SOURCE_DIR}/interfaces/json/
-  OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/interfaces/json/
-)
```

Comment:
We need custom interface for RDK use cases. Please keep it.

Reply:
agree. @svdmaar can we add this.